### ISSUE TITLE
Add docs to Aurelia intellisense

### ISSUE THREAD

**Issue by pkkummermo** [aurelia/vscode-extension#118]
**I'm submitting a feature request**

I'm loving the intellisense feature, and would love for it to be improved with relevant documentation for the custom-element and the props.

For the custom element:
```ts
/**
 * tn-button is a custom button used for CTA and general use throughout the app
 */
@autoinject
export class TnButton extends TnCustomComponent { ...
```` 

![image](https://user-images.githubusercontent.com/1799456/74628220-62360980-5155-11ea-8d73-6f0902622651.png)

For the prop:
```ts
  /**
   * Loading determines whether a button is in a loading state or not.
   */
  @bindable loading: boolean = false;
```

![image](https://user-images.githubusercontent.com/1799456/74628194-4b8fb280-5155-11ea-9ce9-ea100d6cd9e5.png)

Again thank you for your immense great work! The intellisense is a killer feature (while a bit unstable, but such is to be expected for such a complex and young feature)

**Comment by hiaux0** [comment]
Hey, great that you like the feature 
> while a bit unstable

That's why try to get tests running first before I continue.
I don't know yet, but the typescript parser should have docs included, so hopefully not too much to do :)

**Comment by pkkummermo** [comment]
I highly appreciate the effort you put into this! As you say, I think it's highly available already through the TypeScript parser (https://github.com/microsoft/TypeScript/wiki/Using-the-Compiler-API#using-the-type-checker)

```ts
documentation: ts.displayPartsToString(symbol.getDocumentationComment(checker)),
```

**Comment by pkkummermo** [comment]
I have something halfway working which basically traverses the `.ts`-files and looks for documentation on exported classes. If none are found, it falls back to the current behaviour.

![image](https://user-images.githubusercontent.com/1799456/77262259-15ce7400-6c95-11ea-8988-962b6b391dd1.png)

**Comment by hiaux0** [comment]
Closed by #122 
Thanks a lot for your great improvments @pkkummermo
